### PR TITLE
improved sponsor section organization

### LINKED ISSUE #10: Improve Sponsor section organization
The Sponsor section is kinda ugly. The icons are really big and take up to much space(there is like one or two icons per row). Check out the way they are organized at pearlhacks.com or hacknc2022.com, which have better organization.

- [x] Group the icons together so that there are more per row and they are slightly smaller, so we save space.

### PR DESCRIPTION
**Pull request by aryasalian**
This resolves task #10  from the Projects tab. I didn't change the sponsor logo images because I wasn't sure exactly where those images should be stored in the repo but I think we should look into the images for Infosys, Deutsche Bank and Kappa Theta Pi: they have a white background. We should have a transparent image for those logos too like the others.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -202,68 +202,48 @@ <h1>Any Questions?</h1>
         <!-- Sponsors -->
         <div class="section sponsors">
             <a class="nav-anchor" id="sponsors"></a>
-            <div class="container">
+            <div class="container" id="sponsor-section">
                 <h1>Sponsors</h1>
+                <!--shifted this up here, similar to pearlhacks and the 2022 hacknc website-->
```
Comment (atoneyd) [inline index.html]: You can remove this comment; any context/reasoning like this can just go in the pull request description.

**Review by atoneyd — CHANGES_REQUESTED** [review]
Looks great! But could you add some styling for mobile using @media in the CSS? the icons look really small and spaced out when you are on your phone. I think they should be a bit bigger/ less spacing between them, and wrapping may help it look better. Whatever you think looks good!

**Comment by atoneyd** [comment]
> We should have a transparent image for those logos too like the others.

True, our sponsors for this year haven't been finalized yet, but once they are there'll be another task to replace the images

**Comment by aryasalian** [comment]
> Looks great! But could you add some styling for mobile using @media in the CSS? the icons look really small and spaced out when you are on your phone. I think they should be a bit bigger/ less spacing between them, and wrapping may help it look better. Whatever you think looks good!

How about now? I've used a flexbox to make it responsive instead of applying @media rules, I hope that's not an issue since we only care about the image resizing. I've removed the viewport dependent dimension styling for all images, they won't shrink with a decrease in viewport size anymore. Instead they'll wrap to the next row. LMK what you think!

**Comment by aryasalian** [comment]
> > We should have a transparent image for those logos too like the others.
> 
> True, our sponsors for this year haven't been finalized yet, but once they are there'll be another task to replace the images

Alright!

**Review by atoneyd — APPROVED** [review]
Thank you, it looks good! I think you forgot to remove the comment on line 207; once you've done that you can merge in this branch

**Comment by aryasalian** [comment]
yepp, all done! thanks!

**Pull request merged**